taxi/validators.py

```diff
@@ -0,0 +1,22 @@
+import re
+
+from django.core.exceptions import ValidationError
+
+
+def validate_license_number(license_number):
+    if len(license_number) != 8:
+        raise ValidationError(
+            "License number must be exactly 8 characters long."
+        )
+
+    if not re.match(r"^[A-Z]{3}", license_number):
+        raise ValidationError(
+            "The first 3 characters must be uppercase letters."
+        )
```

Comment:
The check `re.match(r'^[A-Z]{3}', license_number)` is redundant because the subsequent check `re.match(r'^[A-Z]{3}[0-9]{5}$', license_number)` already ensures the first three characters are uppercase letters. You can remove the first regex check to simplify the code.

Reply:
done